scripts/build-doc

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+/* globals require process */
```

Comment:
It's not. None of these are. I (or someone) needs to put together an eslint config and routine for these files.